src/components/form/head.vue

```diff
@@ -33,34 +33,37 @@ except according to the terms contained in the LICENSE file.
               to render the tabs even if the form does not have a published
               version (in which case canRoute() will return `false`). -->
               <li v-if="rendersFormTabs" :class="formTabClass('')"
-                :title="formTabTitle" role="presentation">
-                <router-link :to="tabPath('')">
+                role="presentation">
+                <router-link :to="tabPath('')"
+                  v-tooltip.aria-describedby="formTabDescription">
```

Comment:
Before, the `<li>` had the `title` attribute, but now I've put the tooltip on the link. I don't think it makes a big difference for users using a mouse, but for screen readers, I think it's helpful for the link to get the `aria-describedby` attribute.